chalice/app.py

```diff
@@ -397,6 +414,30 @@ def json_body(self):
                     raise BadRequestError('Error Parsing JSON')
             return self._json_body
 
+    @property
+    def files(self):
+        content_type = self.headers.get('content-type')
+        if content_type.startswith('multipart'):
```

Comment:
Why is this only checking multipart?  Are there other content types besides multipart/form-data that we're intending to support here?  I didn't see anything in the tests.